www/src/html.js

```diff
@@ -72,6 +72,12 @@ export default class HTML extends React.Component {
             href={`/safari-pinned-tab.svg`}
             color="#5bbad5"
           />
+          <script src="https://use.typekit.net/vol0sfl.js" />
```

Comment:
Was playing around with some serifs from Typekit again; see first bullet point in the PR description. Gyre Schola is also still in.